src/mixer/basetrackplayer.h

```diff
@@ -100,6 +107,10 @@ class BaseTrackPlayerImpl : public BaseTrackPlayer {
     ControlObjectSlave* m_pPreGain;
     ControlObjectSlave* m_pSpeed;
     ControlObjectSlave* m_pPitchAdjust;
+    QScopedPointer<ControlObjectSlave> m_pInputEnabled;
+    QScopedPointer<ControlObjectSlave> m_pPassthroughEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlEnabled;
+    QScopedPointer<ControlObjectSlave> m_pVinylControlStatus;
```

Comment:
>> A scoped pointer here will confuse the reader

> A scoped pointer is a standard engineering practice -- there's really nothing confusing about it.

Yes, there is nothing against Scoped pointers to avoid delete calls. 
But in this case it is confusing, because it is redundant. The reader might think is parenting not enough? Why a scope pointer required here and not elsewhere.  
 
>>  requires extra CPU.

> Are you serious about this?

Not relay, But it is just a fact that this requires some CPU cycles without any gain.